lib/src/Uncertainty/Distribution/KernelMixture.cxx

```diff
@@ -41,7 +41,7 @@ static const Factory<KernelMixture> Factory_KernelMixture;
 /* Default constructor */
 KernelMixture::KernelMixture()
   : ContinuousDistribution()
-  , kernel_()
+  , p_kernel_()
```

Comment:
I guess this should not be left uninitialized, maybe `p_kernel_(Distribution().getImplementation())`

Reply:
indeed